reference/toc.yml

```
@@ -0,0 +1,18 @@
+toc:
+  - toc: security
```

I think we need this?

```suggestion
toc:
  - file: index.md
  - toc: security
```

---

yes!